experimental/perf/scripts/bench_run_log.py

```diff
@@ -2,9 +2,10 @@
 import json
 from datetime import datetime
 
-from setup_helper import generate_random_dict, get_logger
+from setup_helper import get_logger, get_payload
 
 import wandb
+import wandb.data_types
```

Comment:
you don't need to import this one, maybe just use the exposed media classes directly, i think this ideally should be internal

Reply:
removed